txselector/txbatch_test.go

```diff
@@ -0,0 +1,630 @@
+package txselector
+
+import (
+	"math/big"
+	"sort"
+	"testing"
+
+	"github.com/hermeznetwork/hermez-node/common"
+	"github.com/hermeznetwork/tracerr"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTxBatch_prune(t *testing.T) {
+	type want struct {
+		length int
+		err    error
+	}
+	tests := []struct {
+		name string
+		want want
+	}{
+		// TODO: Add test cases.
```

Comment:
Missing tests cases

Reply:
yes, it's missing. Sorry, I forget these two cases. Let me add them today